clearpath_config/sensors/types/cameras.py

```diff
@@ -704,9 +595,9 @@ class TOPICS:
             COLOR_IMAGE: 'color/image',
             COLOR_CAMERA_INFO: 'color/camera_info'
         }
-        RATE = {
-            COLOR_IMAGE: BaseCamera.FPS,
-            COLOR_CAMERA_INFO: BaseCamera.FPS,
+        TYPE = {
```

Comment:
Blackfly topics match the same same topics as `BaseCamera`. Can we get rid of this?